python_scripts/trees.py

```diff
@@ -369,8 +374,9 @@ def classification_criterion(labels):
 # and Gentoo penguins.
 #
 # With entropy, we are able to assess the quality of each partition. However,
-# the ultimate goal is to evaluate the quality of the overall split and thus
-# combine the measures of entropy in each partition into a single statistic.
+# the ultimate goal is to evaluate the quality of the overall split
+# (the node in decision tree)and thus
```

Comment:
I don't think 'the node in decision tree' is needed and it is a little confusing, are we talking about the child nodes?